index.html

```diff
@@ -693,9 +693,20 @@
               </li>
               <li>Determine which <a>payment handlers</a> support any of the
               <a>payment method identifiers</a> given <var>identifiers</var>.
-              For each resulting payment handler, if payment method specific
-              capabilities supplied by the payment handler match those provided
-              by <var>data</var>, the payment handler matches.
+              </li>
+              <li>For each <var>handler</var> of the resulting payment
+              handlers:
+                <ol>
+                  <li>If required by the specification that defines the
+                  <var>handler</var>, <a data-cite=
+                  "!WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+                  <var>data</var> to an IDL value. Rethrow any exceptions.
```

Comment:
> You can't throw an exception from in parallel.

Ah yeah, will reject acceptPromise. 

> It seems unlikely we want to error if the user installs a payment handler which doesn't accept the provided data. 

Sure... but... 

> Maybe if none of their installed handlers do... but that's already covered by the next step.

Ok, but something bad needs to happen with invalid data? I'm confused about where that should go.